packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/BrandDemo/BrandDemo.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Brand } from '@patternfly/react-core';
-import pfLogo from './images/pfLogo.svg';
+const pfLogo = './images/pfLogo.svg';
```

Comment:
Was this change intentional? Perhaps fixing a bug unrelated to the linter?

Reply:
This was throwing an error that the path could not be resolved, and the `Brand` component `src` prop expects a URL string